packages/component/src/providers/Theme/private/Context.ts

```diff
@@ -18,7 +18,7 @@ import { createContext } from 'react';
 const EMPTY_ARRAY = Object.freeze([] as const);
 const EMPTY_OBJECT = Object.freeze({} as const);
 
-export type ContextType = {
+export type ThemeContextType = {
```

Comment:
Renamed as it gets bundled into types as `ContextType$1` otherwise

Reply:
Good to know. We have tons. 🙂